package.json

```diff
@@ -9,7 +9,9 @@
     "start": "next start -p $PORT",
     "lint": "eslint . --ext js,jsx,ts,tsx",
     "lint-fix": "eslint . --fix --ext js,jsx,ts,tsx",
-    "postinstall": "husky install"
+    "postinstall": "husky install",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Change this to 

```
start-storybook -s ./public -p 6006
```

So all the static assets we use in our components are also acccessible by storybook. 
Ref: https://storybook.js.org/docs/riot/configure/images-and-assets

Please test that this works properly by writing a story for a component that uses some static asset.